src/components/offline/OfflineService.js

```diff
@@ -395,6 +420,18 @@
             var layer = layers[i];
             layersIds.push(layer.id);
             layersOpacity.push(layer.invertedOpacity);
+
+            // if the layer is a KML
+            if (gaKml.isKmlLayer(layer) && /http/.test(layer.url)) {
```

Comment:
dummy, of course. Improvement might still be a good idea. Can we use ^ and `://` ?